src/m_misc.c

```diff
@@ -351,10 +351,10 @@ default_t defaults[] = {
   },
 
   {
-    "menu_background",
-    (config_t *) &menu_background, NULL,
-    {background_on}, {background_on,background_dark}, number, ss_gen, wad_no,
-    "draw menu background (0 = on, 1 = off, 2 = dark)"
+    "menu_backdrop",
+    (config_t *) &menu_backdrop, NULL,
+    {MENU_BG_DARK}, {MENU_BG_DARK,MENU_BG_TEXTURE}, number, ss_gen, wad_no,
+    "draw menu backdrop (0 = dark, 1 = off, 2 = texture)"
```

Comment:
I'll take a closer look later but a quick nitpick for now: since the variable is being renamed which means full control over the default value for the next release, let's use the opportunity to be consistent with the order. 0 = off, 1 = dark, 2 = texture (increasing amount of opacity, zero means off, seems intuitive).

Slightly off topic but the automap color menu item being boom -> vanilla -> zdoom is another example of this kind of inconsistency (vanilla -> boom -> zdoom seems more correct).

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1432/commits/c07f616ac98692c1fc25f4db514b6bce83a65ec6

Reply:
Should we change anything about order of the Automap Overlay options?

Reply:
> Should we change anything about order of the Automap Overlay options?

Seems to work as before. Do you mean make Dark Overlay the default?

Reply:
> Slightly off topic but the automap color menu item being boom -> vanilla -> zdoom is another example of this kind of inconsistency (vanilla -> boom -> zdoom seems more correct).

I have been bitten myself by this before, although I have been the one to introduce this order. I only found it natural to sort presets by default -> variant 1 -> variant2, but my brain indeed works like simple -> more complex -> more complex. 😁

Reply:
> simple -> more complex -> more complex

Should I change it here before we forget?

Reply:
Yes, please!

Reply:
> > Should we change anything about order of the Automap Overlay options?
> 
> Oops I overlooked the automap order which complicates things. I think we're stuck with automap "overlay mode" naming for historical reasons.

I don't think we should change this if it affects the user's config. For the menu background we have an excuse - significant menu redesign. If we force more changes on users, it will just be annoying.

Reply:
I'm undecided. There's nothing wrong with either approach.

Reply:
So, what to do? I'm fine with both approaches, too. I guess @ceski-1 should decide this 😄

Reply:
I'm not sure either. When in doubt, go with traditional/vanilla behavior?

Reply:
I keep it as it is.

Is it OK to merge this?

Reply:
Go.